draft-yasskin-http-origin-signed-responses.md

```diff
@@ -530,10 +560,17 @@ there are no non-significant response header fields in the exchange.
       1. The text string "expires" to the integer value of `expires`.
       1. The text string "headers" to the CBOR representation
          ({{cbor-representation}}) of `exchange`'s headers.
-1. If `signature` is `message`'s signature by `main-certificate`'s public key
-   using `signing-alg`, return "potentially-valid" with `exchange` and whichever
-   is present of `certificate-chain` or `ed25519Key`. Otherwise, return
-   "invalid".
+1. If `certUrl` is present and the SHA-256 hash of `main-certificate`'s
+   `cert_data` is not equal to `certSha256` (whose presence was checked when the
+   `Signature` header field was parsed), return "invalid".
+
+   Note that this intentionally differs from TLS 1.3, which signs the entire
+   certificate chain in its Certificate Verify (Section 4.4.3 of
+   {{?I-D.ietf-tls-tls13}}), in order to allow updating the stapled OCSP
+   response without updating signatures at the same time.
+1. If `signature` is `message`'s signature by `publicKey` using `signing-alg`,
+   return "potentially-valid" with `exchange` and whichever is present of
+   `certificate-chain` or `ed25519Key`. Otherwise, return "invalid".
```

Comment:
Nit: There can be more than one valid signature for a given public key and message. I would say perhaps:

If `signature` is a valid signature of `message` by `publicKey` using `signing-alg` [...]